zerver/models.py

```diff
@@ -1116,6 +1123,7 @@ class UserProfile(AbstractBaseUser, PermissionsMixin):
     onboarding_steps: str = models.TextField(default='[]')
 
     zoom_token: Optional[object] = JSONField(default=None, null=True)
+    webex_token: Optional[object] = JSONField(default=None, null=True)
```

Comment:
@andersk what are your thoughts on the data model here -- should we be trying to refactor this to use the same field for both providers in some way?